Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:14-slim
+
+WORKDIR /usr/src/app
+
+COPY Backend/package*.json ./
+
+RUN npm install
+
+COPY Backend/. /usr/src/app/Backend/
+
+EXPOSE 3000
```

Comment:
No need to expose 3000 port
we will just expose the frontend port, backend can be kept internal

Reply:
For Backend image testing i just expose the port it will be removed in the final integration